### PR TITLE
Mu4 xtz

### PR DESCRIPTION
**Pull request by handrok**
Resolves: *(direct link to the issue)*

*(short description of the changes and the motivation to make the changes)*

- [ ] I signed [CLA](https://musescore.org/en/cla)
- [ ] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [ ] I made sure the code compiles on my machine
- [ ] I made sure there are no unnecessary changes in the code
- [ ] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [ ] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [ ] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: src/engraving/libmscore/masterscore.cpp [hunk 1]
```diff
@@ -512,9 +512,9 @@ void MasterScore::setPos(POS pos, Fraction tick)
     // even though tick position might not have changed, layout might have
     // so we should update cursor here
     // however, we must be careful not to call setPos() again while handling posChanged, or recursion results
-    for (Score* s : scoreList()) {
-        emit s->posChanged(pos, unsigned(tick.ticks()));
-    }
+//    for (Score* s : scoreList()) {
+//        emit s->posChanged(pos, unsigned(tick.ticks()));
+//    }
```
Comment (igorkorsukov) [inline src/engraving/libmscore/masterscore.cpp]: переделано

File: src/engraving/libmscore/masterscore.h [hunk 2]
```diff
@@ -76,7 +76,7 @@ class MidiMapping
 
 class MasterScore : public Score
 {
-    Q_OBJECT
+//    Q_OBJECT
```
Comment (igorkorsukov) [inline src/engraving/libmscore/masterscore.h]: переделано

File: src/engraving/libmscore/mscore.cpp [hunk 3]
```diff
@@ -223,8 +227,8 @@ void MScore::init()
 void MScore::registerUiTypes()
 {
 #ifdef SCRIPT_INTERFACE
-    qRegisterMetaType<Spanner::Anchor>("Anchor");
-    qRegisterMetaType<SegmentType>("SegmentType");
+//    qRegisterMetaType<Spanner::Anchor>("Anchor");
+//    qRegisterMetaType<SegmentType>("SegmentType");
```
Comment (igorkorsukov) [inline src/engraving/libmscore/mscore.cpp:231]: нужно отключить SCRIPT_INTERFACE (не дефайнить)

File: src/engraving/libmscore/masterscore.cpp (hunk 1 above)
Comment (handrok) [reply src/engraving/libmscore/masterscore.cpp]: ага, форк слегка отстает

**Pull request closed without merging**